src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -74,21 +74,45 @@ tests:
                         EpochKey2: "hex:d2d1d2d3d4d5d6d7d8d9dadbdcdddedf",
```

Comment:
Is there a change to test-plan spec PR for the changes you are making here?

Will you be removing the old step#1

Reply:
not yet - waiting to hear back from the QA team about whether this is OK for them. Works for me, but I'm not going to do the TP until they're good.

Need both steps now, so I suppose I should call them step 1a and step 1b

Reply:
Test plan change here: https://github.com/CHIP-Specifications/chip-test-plans/pull/3754